src/core/symbology/qgslinesymbollayer.cpp

```diff
@@ -1290,6 +1291,53 @@ void QgsTemplatedLineSymbolLayerBase::setPlacement( Qgis::MarkerLinePlacement pl
 
 QgsTemplatedLineSymbolLayerBase::~QgsTemplatedLineSymbolLayerBase() = default;
 
+
+QgsTemplatedLineSymbolLayerBase::BlankAreas QgsTemplatedLineSymbolLayerBase::parseBlankAreas( const QString &strBlankArea, const QgsRenderContext &rc, Qgis::RenderUnit unit, int partNum, int iRing )
+{
+  BlankAreas blankAreas;
+
+  // We reuse Wkt MultiPolygon format to define distances tuple (start, end) for polygon/line with
+  // possible parts and rings
+  QgsMultiPolygon polygon;
```

Comment:
I don't understand this -- area we dealing with blank areas, or blank start/end distances??